app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,40 @@
+<resources>
+    <string name="main_activity_description">Chat with your mentors.</string>
+
+    <string name="login">Login</string>
+    <string name="chat_with_mentor">Chat with a mentor</string>
+
+    <!-- NOTIFICATION SERVICE -->
+    <string name="notification_title">You have a new notification</string>
+
+    <!-- MAIN ACTIVITY -->
+    <string name="error_not_connected">You must first login!</string>
+    <string name="error_authentication_canceled">Login aborted</string>
```

Comment:
Is "aborted" the general term used in these kind of app error messages? The source literally translates to "Authentication cancelled", is there something closer to that that is commonly used?

Reply:
I think aborted is okay, this is a popup message to 'show the feature is working' rather than a permanent piece of copy for in-app use